src/util/locking/scripts/RedisLuaScripts.ts

```diff
@@ -0,0 +1,151 @@
+import type { Callback, Redis as RedisInterface } from 'ioredis';
```

Comment:
Why is `Redis` being renamed here?

Reply:
Remnant, should be able to get rid of it.